### ISSUE TITLE
The simpliest way to implement messagesencrypted

### ISSUE THREAD

**Issue by g1itch** [Bitmessage/PyBitmessage#1551]
Hello!

It seems the 'messagesencrypted' setting may be implemented via [sqlcipher](../../../sqlcipher/sqlcipher) using [pysqlcipher](https://pypi.org/project/pysqlcipher/).

**Comment by PeterSurda** [comment]
I agree, I have been postponing this because many distros don't bundle pysqlcipher and it would have to be installed though pip.

**Comment by dimyme** [comment]
as long as  --test /  checkdepends.py / depends.py / setup.py takes proper care of that it would be OK to introduce a new dependency.

**Comment by sgj3** [comment]
Can someone refer me to documentation about "messagesencrypted"?  I am interested in implementing this functionality, which I assume means that all messages will be stored encrypted on disk.
